src/services/API/Plio.js

```diff
@@ -40,9 +40,25 @@ export default {
     });
   },
 
-  getAllPlios() {
-    // returns all the plios created by the user
-    return apiClient().get(pliosEndpoint);
+  getAllPlios(userId = null) {
+    // returns all the plios created by the user if userId is present
+    // else returns all the plios
+    if (userId == null) return apiClient().get(pliosEndpoint);
+    else
+      return apiClient().get(pliosEndpoint, {
+        params: { user: userId },
+      });
+  },
+
+  getAllPlioIds(userId = null) {
```

Comment:
I think the point here was to avoid fetching all the data when we only need the UUID. Maybe we can say pass a param in the `getAllPlios` API call to just return the given set of keys. @deepansh96 ?